cityofaustin/atd-vz-data
# Crash details page: crash map

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17182

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Follow instructions in https://github.com/cityofaustin/atd-vz-data/pull/1440 to start the data model collector branch and run the CRIS import ETL, or if you already have it set up use that!
2. Go to a crash, see that the crash map says "Geocode Provider: TxDOT CRIS"
3. Edit the crash coordinates, move it to some other road. See the location id change and it will now say "Geocode Provider: Manual Q/A".


---
#### Ship list
